Makefile

```diff
@@ -130,10 +130,10 @@ HELM_ARCHIVE=helm-$(HELM_VERSION)-$(OS)-$(EXEC_ARCH).tar.gz
 HELM_ARCHIVE_BASE=$(OS)-$(EXEC_ARCH)
 
 # spark
-export SPARK_VERSION=3.3.1
+export SPARK_VERSION=3.3.3
 export SPARK_HOME=$(BASE_DIR)$(TOOLS_DIR)/spark
 export SPARK_SUBMIT_CMD=$(SPARK_HOME)/bin/spark-submit
-export SPARK_PYTHON_IMAGE=docker.io/apache/spark-py:v$(SPARK_VERSION)
+export SPARK_PYTHON_IMAGE=docker.io/apache/spark-py:v3.3.1
```

Comment:
Can we not hard code this but have two variables defined and explain that they should be the same minor version at least. Patch versions may differ based on availability.